pkg/update/update.go

```diff
@@ -372,7 +372,7 @@ func (u *runnerImpl) Start(ctx context.Context, options ...compose.StartOption)
 func (u *runnerImpl) Cancel(ctx context.Context) error {
 	return u.store.lock(func(db *session) error {
 		if !u.State.IsOneOf(StateCreated, StateInitializing, StateInitialized,
-			StateFetching, StateFetched, StateInstalling, StateInstalled) {
+			StateFetching, StateFetched, StateInstalling, StateInstalled, StateCompleting) {
```

Comment:
This will allow users to "cancel" an update that has been successfully started, as a result successful update is considered as "failing".

Maybe we rather should fix the "composectl update complete" implementation, so it guarantees moving the update state to the "completed" as long as the update is successfully started.